v21.2/ui-transactions-page.md

```diff
@@ -64,14 +66,14 @@ Significant transactions on your database are likely to have a high execution co
 
 ### Time interval
 
-By default, the Transactions page displays all transactions executed within a one-hour time interval. The display is cleared at the end of each interval. You can change the interval with the [`diagnostics.reporting.interval`](cluster-settings.html#settings) cluster setting.
+The Transactions page aggregates all transactions executed within a configured time interval. The default interval is one hour. You can change the interval with the [`diagnostics.reporting.interval`](cluster-settings.html#settings) [cluster setting](set-cluster-setting.html). The interval start time for each transaction fingerprint is displayed in the Interval Start Time column.
```

Comment:
Hi Stephanie, we can remove this section now for 21.2. The 'diagnostics.reporting.interval' no longer configures the interval/flush interval for persisted stats.

Reply:
We can remove the Time Interval in section Statements page as well.